### DISCUSSION TITLE
Collaboration with Anaconda and the PyScript project

### DISCUSSION THREAD

**Discussion by gabrielfougeron** [pyodide/pyodide#2733] [General]
I've recently learned about PyScript https://www.anaconda.com/blog/pyscript-python-in-the-browser. As I understand it, this is an additional layer on top of Pyodide developed by Anaconda. I'm assuming the Pyodide was involved to some extend as well.

Would you consider (maybe in the form of a blog post) clarifying a few points for people like me who are interested? Namely:
- What has your involvement been so far / what will it likely be in the future?
- What are the differences between Pyodide and PyScript for someone who wants to "use python in the browser"? How should I choose between the two?
- Is there a known roadmap for Pyscript that you can communicate on?
- To what extend is PyScript a "good" news for Pyodide (more visibility? more ressources?) ? Are there less good side effects to be expected from Anaconda's involvement?

**Comment by hoodmane** [comment]
Sounds like a good idea! Thanks @gabrielfougeron.
